crates/rpc/rpc/src/eth/filter.rs

```diff
@@ -55,6 +56,9 @@ where
     }
 }
 
+/// Threshold for deciding between cached and range mode processing
+const CACHED_MODE_BLOCK_THRESHOLD: u64 = 100;
```

Comment:
after thinking about this, I think we can increase this a bit, let's do 250, which would be ~8min on opstack and almost an hr on mainnet